### ISSUE TITLE
FindMKL: ability to use PGI compatibility libraries

### ISSUE THREAD

**Issue by multiplemonomials** [Amber-MD/cmake-buildscripts#12]
Need to find a system that has these installed.  SDSC Comet?

**Comment by agoetz** [comment]
@multiplemonomials We have PGI compilers on comet. You can load these as a module
```
module purge
module load gnutools
module load cmake
module load pgi
```
We don't have mpi installed with pgi compilers, though.

**Comment by agoetz** [comment]
@multiplemonomials In case you feel like testing, we have an older version of PGI compilers with MPI support on Comet:
```
module purge
module load gnutools 
export MODULEPATH=/share/apps/compute/pgi/modulefiles:$MODULEPATH
module load pgi/16.10
module load openmpi/1.10.2
```

**Comment by multiplemonomials** [comment]
Should be fixed by d80b76097
